client/messaging/message-input.tsx

```diff
@@ -108,11 +108,10 @@ export const MessageInput = React.forwardRef<MessageInputHandle, MessageInputPro
         className={props.className}
         label='Send a message'
         value={message}
-        maxLength={500}
         floatingLabel={false}
         allowErrors={false}
         showDivider={props.showDivider}
-        inputProps={{ autoComplete: 'off' }}
+        inputProps={{ autoComplete: 'off', maxLength: 500 }}
```

Comment:
Honestly I'm not really a fan of adding this, even to stuff that has a max length. I prefer a validation that doesn't actively limit the amount of text that can be put into the field. If you e.g. paste text longer than 500 characters into this field with this attribute, it will be abruptly cut off at that point. If you paste it into a field that merely validates that length, the full text will get pasted and you have the opportunity to edit it to get it below the length, instead of just having it truncated at an arbitrary spot

Reply:
Yeah, that makes sense. I'll just remove the length limit on the client for now, but will add a proper validation in a later PR.